packages/server/graphql/public/mutations/updateTeamPromptResponse.ts

```diff
@@ -0,0 +1,57 @@
+import {JSONContent} from '@tiptap/core'
+import {SubscriptionChannel} from 'parabol-client/types/constEnums'
+import {getTeamPromptResponseById} from '../../../postgres/queries/getTeamPromptResponsesByIds'
+import {updateTeamPromptResponseContentById} from '../../../postgres/queries/updateTeamPromptResponseContentById'
+import {getUserId, isTeamMember} from '../../../utils/authorization'
+import publish from '../../../utils/publish'
+import standardError from '../../../utils/standardError'
+import extractTextFromTipTapJSONContent from '../../mutations/helpers/tiptap/extractTextFromTipTapJSONContent'
+import {MutationResolvers} from '../resolverTypes'
+
+const updateTeamPromptResponse: MutationResolvers['updateTeamPromptResponse'] = async (
+  _source,
+  {updatedTeamPromptResponse},
+  {authToken, dataLoader, socketId: mutatorId}
+) => {
+  const viewerId = getUserId(authToken)
+  const operationId = dataLoader.share()
+  const subOptions = {mutatorId, operationId}
+  const {teamPromptResponseId, content} = updatedTeamPromptResponse
+
+  // AUTH
+  const promptResponse = await getTeamPromptResponseById(teamPromptResponseId)
```

Comment:
+1 if we use `dataLoader` here, we might have one less DB call when resolving the response. I see you're already updating the `promptResponse` after the insert so just changing this line with `dataLoader` will do the job